internal/tableland/impl/mesa.go

```diff
@@ -117,6 +117,11 @@ func (t *TablelandMesa) RelayWriteQuery(
 		return tableland.RelayWriteQueryResponse{}, fmt.Errorf("chain id %d isn't supported in the validator", chainID)
 	}
 
+	if !stack.AllowTransactionRelay {
```

Comment:
Blocks `RelayWriteQuery` call on a specific chain if the flag is set to false on that chain